test_files/structs.d

```diff
@@ -26,28 +26,26 @@ struct A
 
 struct C
 {
-    struct
+    struct _Anonymous_0
     {
         int x;
         int y;
     }
```

Comment:
Fixed, however there is an issue with translation of real anonymous structs (the ones that do no have an associated variable).